edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/ConfigUpdater.cs

```diff
@@ -37,7 +42,9 @@ public async Task Init(IConfigSource configProvider)
             Preconditions.CheckNotNull(configProvider, nameof(configProvider));
             try
             {
-                configProvider.SetConfigUpdatedCallback(this.HandleUpdateConfig);
+                configProvider.ConfigUpdates +=
```

Comment:
the downside of Event vs Callback is that async event will be "fire and forget". Do you see any issues with that?